examples/lora/sensor-receive/Makefile

```diff
@@ -0,0 +1,17 @@
+# Makefile for user application
+
+# Specify this directory relative to the current application.
+TOCK_USERLAND_BASE_DIR = ../../../
+
+# Which files to compile.
+CXX_SRCS := $(wildcard *.cc)
+
+# Use the libtock-c Make system
+EXTERN_LIBS := $(TOCK_USERLAND_BASE_DIR)/libradio/RadioLib
+include $(TOCK_USERLAND_BASE_DIR)/libradio/Makefile
+
+# `EXTERN_LIBS` points to `libradio/RadioLib` so we pull in `Makefile.app`
+# manually.
+include $(TOCK_USERLAND_BASE_DIR)/libradio/Makefile.app
```

Comment:
`EXTERN_LIBS` can be a space separated list if-needed, though, as-mentioned above I don't think it's needed at all in this case